drivers/sensor/bh1750/bh1750.c

```diff
@@ -0,0 +1,140 @@
+/* bh1750.c - Driver for BH1750 ambient light sensor */
+/*
+ * Copyright (c) 2023 Sngular People SL.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT rohm_bh1750
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util_macro.h>
+
+#include "bh1750.h"
+
+LOG_MODULE_REGISTER(BH1750, CONFIG_SENSOR_LOG_LEVEL);
+
+static int bh1750_channel_get(const struct device *dev,
+			       enum sensor_channel chan,
+			       struct sensor_value *val)
+{
+	struct bh1750_data *data = dev->data;
+	
+	if (chan != SENSOR_CHAN_LIGHT) {
+		return -ENOTSUP;
+	}
+	
+	val->val1 = (int32_t)(data->ambient_light);
+	val->val2 = (data->ambient_light - val->val1) * 1000000;
+
+	return 0;
+}
+
+
+static int bh1750_sample_fetch(const struct device *dev,
+				enum sensor_channel chan)
+{
+	struct bh1750_data *drv_data = dev->data;
+	const struct bh1750_config *cfg = dev->config;
+	uint8_t buf[2];
+	int rc = 0;
+
+	// In case of one time measurement we need to start a new measurement
+#ifdef CONFIG_BH1750_MODE_ONE_TIME
+	uint8_t buf_ctrl = BH1750_CTRL_MEAS;	
+	rc = i2c_write_dt(&cfg->i2c, &buf_ctrl, 1);
+	if(rc < 0){
+		LOG_ERR("Error writing in i2c bus: %d", rc);
+		return -ENXIO;
+	} else {
+		LOG_DBG("Starting a new measurement: %d", buf_ctrl);
+	}
+
+	// We need to wait for the new measurement to be taken
+	// According to datasheet page 5 section "Measurement mode explanation"
+	// 120 ms for H_RESOLUTION_MODE and H_RESOLUTION_MODE2
+	// 16 ms for L_RESOLUTION_MODE
```

Comment:
Use `/* */` comments. Please fix throughout the driver

Reply:
Ok, done!